generator/views/funnel_analysis_view.py

```diff
@@ -0,0 +1,203 @@
+        ]
+        return [
+            {
+                "name": "funnel_analysis",
+                "extends": ["events_daily_table"],
+                "dimensions": dimensions,
+                "measures": count_measures + fractional_measures,
+            }
+        ]
+
+    def _event_types_lookml(self) -> List[Dict[str, Any]]:
+        events = (
+            [
+                {
+                    "name": "event_types",
+                    "derived_table": {
+                        "sql": (
+                            "SELECT "
+                            "mozfun.event_analysis.aggregate_match_strings( "
+                            "ARRAY_AGG( "
+                            "mozfun.event_analysis.event_index_to_match_string(index))) AS match_string "
+                            "FROM "
+                            f"{self.tables[0]['event_types']} "
+                            "WHERE "
+                            "{% condition category %} category {% endcondition %} "
+                            "AND {% condition event %} event {% endcondition %}"
+                        )
+                    },
+                    "filters": [
+                        {
+                            "name": "category",
+                            "type": "string",
+                            "suggest_explore": "event_names",
+                            "suggest_dimension": "event_names.category",
+                        },
+                        {
+                            "name": "event",
+                            "type": "string",
+                            "suggest_explore": "event_names",
+                            "suggest_dimension": "event_names.event",
+                        },
+                    ],
+                    "dimensions": [
+                        {
+                            "name": "match_string",
+                            "hidden": "yes",
+                            "sql": "${TABLE}.match_string",
+                        }
+                    ],
+                }
+            ]
+            + [
+                {
+                    "name": f"event_type_{n}",
+                    "extends": ["event_types"],
+                }
+                for n in range(1, self.n_events() + 1)
+            ]
+            + [
+                {
+                    "name": "event_names",
+                    "derived_table": {
+                        "sql": (
+                            "SELECT category, "
+                            "  event, "
+                            "  property.key AS property_name, "
+                            "  property_value.key AS property_value, "
+                            f"FROM {self.tables[0]['event_types']} "
+                            "LEFT JOIN UNNEST(event_properties) AS property "
+                            "LEFT JOIN UNNEST(property.value) AS property_value "
+                        )
+                    },
+                    "dimensions": [
```

Comment:
Can you add descriptions for these?

Reply:
Good call!

Reply:
I think these still need to be added :)

Reply:
Ah, this view doesn't need descriptions. It's only used for filter suggestions.